### ISSUE TITLE
Research Esri Road Closure Feed Solutions

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#5176]
@jaime-mckeown asking you to take the lead on this and just included in an email to setup a meeting with Esri.

Esri has a [road closure solution](https://doc.arcgis.com/en/arcgis-solutions/reference/introduction-to-road-closures.htm) that may allow us to easily accomplish [#5154](https://app.zenhub.com/workspace/o/cityofaustin/atd-data-tech/issues/5154).

To be determined:
- Does this tool work? How do we "install" it in our ArcGIS Online org?
- Are navigation providers able to consume the feed from this tool?
- Does Esri have (or plan to have) any support for the [Work Zone Data Specification](https://github.com/usdot-jpo-ode/jpo-wzdx)?

**Comment by johnclary** [comment]
FYI @TracyLinder

**Comment by jaime-mckeown** [comment]
Meeting with Zena from Esri on 2/4 @ 4pm.

**Comment by jaime-mckeown** [comment]
- We are looking for a daily update that we can share with Nav companies.
- John looked at it 3 years ago, but it has changed since then.
- Most users use this program to focus on Emergency Management.
- Application is easily updated and changed, they are templates so you don't have to be a Developer to use and update the software.
- Standard templates and icons are readily available in the App. But they can be changed
- ArcGIS Solutions website has been revamped, Road Closures has info in this website.
- There is a one-click button to deploy the solution in AGOL. Feature services, maps, apps, dashboard all are deployed. You can use all or choose the ones you want to use.
- Zena to send links to Try it Now so we can test it out before making a decision.
- It can be customized, logos can change, etc.
- Web App Builder is used to configure the information on the right side. Questions can be changed, they can be set as required.
- Configuration can be changed to suit our needs, can be added to, etc.
- WAZE is a partner company to Esri.
- Connected citizens program has different levels. WAZE will ingest the feature service from Esri. Online portal to send feature services. There are options for how to share with companies like WAZE. Real-time feed.

Zena will be sending some links to some Road Closure Solutions that have been deployed.

**Comment by johnclary** [comment]
@jaime-mckeown would you reach out to Jen D. and see if we have a Waze connected community contact? We need to clarify if Waze would consume an AGOL feature service from the Esri Road Closures solution.

**Comment by jaime-mckeown** [comment]
Emailed Jen regarding the comment above on 2/11.

**Comment by jaime-mckeown** [comment]
Installed Road Closures, looking into Solution.

**Comment by jaime-mckeown** [comment]
@johnclary see notes below.
- There is a specific section regarding sharing data with Waze under Road Closures:
   https://doc.arcgis.com/en/arcgis-solutions/reference/configure-road-closures.htm
- I checked with Thomas and the City of Austin's Hub is configured to share Open Data already. I wanted to double check since the Organizational Admin must configure this setting.
- Regarding assigning a Creative Commons License, I'm not familiar with that piece, but I assume is part of sharing data via the Open Data Portal.
- I checked with Lance in the MMC and our Waze Connected Community is basically the same thing as the Waze for Cities Program mentioned in the link above. He said they just rebranded the name. He does not have a direct contact with Waze, but said he would help me to get logged into Waze for Cities so I can see if there is any contact info in the application.
- The bottom of the page in the link above gives directions for configuring the layer for sharing with Waze.
